app/views/general/_responsive_topnav.html.erb

```diff
@@ -1,38 +1,8 @@
-<div id="topnav" class="topnav">
+<div id="topnav" class="topnav <%= @pro_area ? 'pro' : '' %>">
```

Comment:
Separately, @wrightmartin has added an `in_pro_area` class to the body too, it seems like that would serve for all of the cases you have here, so perhaps we should standardise on that (naming aside)?